packages/clarity-js/src/layout/node.ts

```diff
@@ -104,6 +105,20 @@ export default function (node: Node, source: Source): Node {
                     break;
                 case "NOSCRIPT":
                 case "META":
```

Comment:
Let's talk offline about this - since this is a lot of extra code we just introduced.

Reply:
Synced offline - will revisit this in future iterations.